frontend/pages/packs/ManagePacksPage/ManagePacksPage.tsx

```diff
@@ -0,0 +1,207 @@
+            "error",
+            `Unable to remove ${packOrPacks}. Please try again.`
+          )
+        );
+        toggleRemovePackModal();
+      });
+  }, [dispatch, selectedPackIds, toggleRemovePackModal]);
+
+  const onEnableDisablePackSubmit = useCallback(
+    (selectedTablePackIds: any, disablePack: boolean) => {
+      const packOrPacks = selectedPackIds.length === 1 ? "pack" : "packs";
+      const enableOrDisable = disablePack ? "disabled" : "enabled";
+
+      console.log("selectedPackIds:", selectedPackIds);
+
+      const promises = selectedTablePackIds.map((id: number) => {
+        console.log("id:", id);
+        console.log("disablePack:", disablePack);
+        return dispatch(packActions.update({ id }, { disabled: disablePack }));
+      });
+
+      return Promise.all(promises)
+        .then(() => {
+          dispatch(
+            renderFlash(
+              "success",
+              `Successfully ${enableOrDisable} selected ${packOrPacks}.`
+            )
+          );
+          dispatch(packActions.loadAll());
+        })
+        .catch(() => {
+          dispatch(
+            renderFlash(
+              "error",
+              `Unable to ${enableOrDisable} selected ${packOrPacks}. Please try again.`
+            )
+          );
+        });
+    },
+    [dispatch, selectedPackIds]
+  );
+
+  const onEnablePackClick = (selectedTablePackIds: any) => {
+    setSelectedPackIds(selectedTablePackIds);
+    onEnableDisablePackSubmit(selectedTablePackIds, false);
+  };
+
+  const onDisablePackClick = (selectedTablePackIds: any) => {
+    setSelectedPackIds(selectedTablePackIds);
+    onEnableDisablePackSubmit(selectedTablePackIds, true);
+  };
+
+  return (
+    <div className={baseClass}>
+      <div className={`${baseClass}__wrapper body-wrap`}>
+        <div className={`${baseClass}__header-wrap`}>
+          <div className={`${baseClass}__header`}>
+            <div className={`${baseClass}__text`}>
+              <h1 className={`${baseClass}__title`}>
+                <span>Packs</span>
+              </h1>
+              <div className={`${baseClass}__description`}>
+                <p>
+                  Manage query packs to schedule recurring queries for your
+                  hosts.
+                </p>
+              </div>
+            </div>
+          </div>
+          {!isOnlyObserver && (
+            <div className={`${baseClass}__action-button-container`}>
+              <Button
+                variant="brand"
+                className={`${baseClass}__create-button`}
+                onClick={onCreatePackClick}
+              >
+                Create new pack
+              </Button>
+            </div>
+          )}
+        </div>
+        <div>
+          {true &&
+            renderTable(
+              onRemovePackClick,
+              onEnablePackClick,
+              onDisablePackClick,
+              packsList,
+              packsErrors
+            )}
+        </div>
```

Comment:
I think this could be written as:
```
{renderTable(
  onRemovePackClick,
  onEnablePackClick,
  onDisablePackClick,
  packsList,
  packsErrors
)}
```
Does that work?